### DISCUSSION TITLE
Amasty Shopby - show more attributes

### DISCUSSION THREAD

**Discussion by danielciornei** [breezefront/community#307] [1. Help]
Hello.

we are running the latest version of swissup/module-breeze-amasty-shopby and we noticed some issues recently with the show more attributes. the feature has stoped working 

```
<item name="Amasty_Shopby/js/components/amShopbyFilterHideMoreOptions" xsi:type="array">
                                <item name="path" xsi:type="string">Swissup_BreezeAmastyShopby/js/components/amShopbyFilterHideMoreOptions</item>
      </item>
```

Thank you

**Comment by danielciornei** [comment]
we manage to solve it by adding auto load 

`<item name="Amasty_Shopby/js/components/amShopbyFilterHideMoreOptions" xsi:type="array">
     <item name="path" xsi:type="string">Swissup_BreezeAmastyShopby/js/components/amShopbyFilterHideMoreOptions</item>
     <item name="autoload" xsi:type="boolean">true</item>
</item>
`

and changing the bundle from dynamic to default

`<item name="default" xsi:type="array">
        <item name="items" xsi:type="array">
               <item name="amShopbyHelpers" xsi:type="array">
`

is there a better way to achieve working more-less filters?

**Comment by danielciornei** [comment]
looks like the above fix was creating problems and solution was simpler 

replacing the below fixed the issue with less/more links on our site. implemented using a composer patch and had no problems since


```
-                            <item name="Amasty_Shopby/js/components/amShopbyFilterHideMoreOptions" xsi:type="array">
+                            <item name="amShopbyFilterHideMoreOptions" xsi:type="array">
```

**Comment by itsrcrowell** [comment]
@danielciornei  I've just created this [PR](https://github.com/breezefront/module-breeze-amasty-shopby/pull/1) to fix this issue
